server/store/session.go

```diff
@@ -144,6 +146,33 @@ func (s sessionStore) ListForApp(appID apps.AppID) ([]*model.Session, error) {
 	return ret, nil
 }
 
+func (s sessionStore) ListUsersWithSessions(appID apps.AppID) ([]string, error) {
+	userKeys := make([]string, 0)
+
+	for i := 0; ; i++ {
```

Comment:
Since we're not using `i`

```suggestion
	for {
```

Reply:
What does this loop do for us? It seems like the exact same operation is done on each iteration

Reply:
yep, that doesn't feel right.